std/experimental/allocator/package.d

```diff
@@ -605,6 +605,8 @@ unittest
     assert(a == [0, 0, 0]);
 }
 
+import std.experimental.allocator.gc_allocator: isGCSafeAllocator;
```

Comment:
https://issues.dlang.org/show_bug.cgi?id=314

Importing it selectively makes it public with no way to make it private.